[2025-03-02T00:15:26.890+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-02T00:15:26.902+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: AlphaVantage_Stock_Price_Prediction_Analysis.load scheduled__2025-02-28T00:15:00+00:00 [queued]>
[2025-03-02T00:15:26.910+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: AlphaVantage_Stock_Price_Prediction_Analysis.load scheduled__2025-02-28T00:15:00+00:00 [queued]>
[2025-03-02T00:15:26.911+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-02T00:15:26.916+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2025-02-28 00:15:00+00:00
[2025-03-02T00:15:26.925+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=239) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-02T00:15:26.925+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'AlphaVantage_Stock_Price_Prediction_Analysis', 'load', 'scheduled__2025-02-28T00:15:00+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpm06v5g7l']
[2025-03-02T00:15:26.927+0000] {standard_task_runner.py:105} INFO - Job 58: Subtask load
[2025-03-02T00:15:26.926+0000] {standard_task_runner.py:72} INFO - Started process 248 to run task
[2025-03-02T00:15:26.961+0000] {task_command.py:467} INFO - Running <TaskInstance: AlphaVantage_Stock_Price_Prediction_Analysis.load scheduled__2025-02-28T00:15:00+00:00 [running]> on host b90f03c6f926
[2025-03-02T00:15:27.019+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='AlphaVantage_Stock_Price_Prediction_Analysis' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T00:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T00:15:00+00:00'
[2025-03-02T00:15:27.020+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-02T00:15:27.431+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T00:15:27.859+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T00:17:13.092+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-02T00:17:13.098+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-02T00:17:13.121+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-02T00:17:13.122+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=AlphaVantage_Stock_Price_Prediction_Analysis, task_id=load, run_id=scheduled__2025-02-28T00:15:00+00:00, execution_date=20250228T001500, start_date=20250302T001526, end_date=20250302T001713
[2025-03-02T00:17:13.149+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-02T00:17:13.160+0000] {taskinstance.py:3925} ERROR - Error scheduling downstream tasks. Skipping it as this is entirely optional optimisation. There might be various reasons for it, please take a look at the stack trace to figure out if the root cause can be diagnosed and fixed. See the issue https://github.com/apache/***/issues/39717 for details and an example problem. If you would like to get help in solving root cause, open discussion with all details with your managed service support or in Airflow repository.
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3921, in schedule_downstream_tasks
    return TaskInstance._schedule_downstream_tasks(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3870, in _schedule_downstream_tasks
    partial_dag = task.dag.partial_subset(
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 2664, in partial_subset
    t.task_id: _deepcopy_task(t)
               ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 2661, in _deepcopy_task
    return copy.deepcopy(t, memo)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 143, in deepcopy
    y = copier(memo)
        ^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 1388, in __deepcopy__
    setattr(result, k, copy.deepcopy(v, memo))
                       ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 136, in deepcopy
    y = copier(x, memo)
        ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 221, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
                             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 136, in deepcopy
    y = copier(x, memo)
        ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 201, in _deepcopy_tuple
    y = [deepcopy(a, memo) for a in x]
         ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 162, in deepcopy
    y = _reconstruct(x, memo, *rv)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 259, in _reconstruct
    state = deepcopy(state, memo)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 136, in deepcopy
    y = copier(x, memo)
        ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 221, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
                             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 162, in deepcopy
    y = _reconstruct(x, memo, *rv)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 259, in _reconstruct
    state = deepcopy(state, memo)
            ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 136, in deepcopy
    y = copier(x, memo)
        ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 221, in _deepcopy_dict
    y[deepcopy(key, memo)] = deepcopy(value, memo)
                             ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/copy.py", line 151, in deepcopy
    rv = reductor(4)
         ^^^^^^^^^^^
TypeError: cannot pickle '_thread.lock' object
[2025-03-02T00:17:13.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
